TwilioVerifySDK/TwilioVerify/Sources/API/ChallengeAPIClient.swift

```diff
@@ -42,7 +42,7 @@ extension ChallengeAPIClient: ChallengeAPIClientProtocol {
   func get(withSid sid: String, withFactor factor: Factor, success: @escaping SuccessResponseBlock, failure: @escaping FailureBlock) {
     func getChallenge(retries: Int = BaseAPIClient.Constants.retryTimes) {
       do {
-        let authToken = try authentication.generateJWT(forFactor: factor)
+        let authToken = try authentication.generateJWT(forFactor: factor, allowIphoneMigration: false)
```

Comment:
What about storing `allowIphoneMigration` as a property of the `factor`, so we can get the defined value (allow or not) from the storage?